TTHAnalysis/python/plotter/ttH-multilepton/lepchoice-ttH-FO.txt

```diff
@@ -21,7 +21,7 @@ LepGood4_(\w+) : LepGood_\1[iLepFO_Recl[3]]; AlsoData
 nLepFO\b : nLepFO_Recl; AlsoData
 nLepTight\b : nLepTight_Recl; AlsoData
 nTauFO\b : nTauSel_Recl; AlsoData
-nTauTight\b : nTauSel_Recl_Tight; AlsoData
```

Comment:
This is a modification that we don't commit.

There is a friend treewith that variable, we can either use that or maintain the change but only in the working directory (i.e. non committing it), as it would break the code for everyone else.

Reply:
(i.e. please undo the change)

Reply:
reverted